about.html

```diff
@@ -0,0 +1,161 @@
+            </g>
+          </svg>
+        </div>
+        <div class="copyright-text">
+          <p class="fw-8">CC Global</p>
+          <p class="fw-8">Summit 2015</p>
+          <p class="fw-5 text-dbar">14th - 17th Oct 2015 | Seoul, Korea</p>
+        </div>
+      </div>
+    </section>
+    <section class="prev-summits">
+      <h2>See the past CC Global Summits</h2>
+      <p class="text-small">Take a look at the last two cCC Global Summits which took place in Buenons Aires and Awrsaw
+      </p>
+      <div class="flex flex-wrap flex-center flex-gap-1">
+        <div class="image">
+          <img
+            src="https://images.unsplash.com/photo-1501386761578-eac5c94b800a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1050&q=80"
+            alt="concert-2016">
+          <div class="image-text flex-column flex-center">
+            <p class="heading">2013</p>
+            <p class="description">CC Global Summit 2011 in Warsaw</p>
+          </div>
+        </div>
+        <div class="image">
+          <img
+            src="https://images.unsplash.com/photo-1559930198-26e8d7f0a4f7?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80"
+            alt="concert-2015">
+          <div class="image-text flex-column flex-center">
+            <p class="heading">2011</p>
+            <p class="description">CC Global Summit 2011 in Warsaw</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="partner">
+      <h2>Partner</h2>
+      <ul class="flex flex-wrap flex-center">
+        <li><img src="./images/icons/mozilla.svg" alt="mozilla-partner"></li>
+        <li><img src="https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png" alt="google-partner"></li>
+        <li><img src="./images/icons/naver.svg" alt="naver-partner"></li>
+        <li><img src="https://media-prd.coachella.com/templates/template10/svgs/coachella-logo.svg" alt="coachella-partner"></li>
+        <li><img src="./images/icons/airbnb.svg" alt="airbnb-partner"></li>
+      </ul>
+    </section>
+  </main>
+  <footer>
```

Comment:
**Footer**
- It contains a logo aligned to the left and text aligned to the right.
The template:
![Screen Shot 2021-09-07 at 19 01 05](https://user-images.githubusercontent.com/22258274/132424848-830f78f7-248a-470b-be4f-0d84754f8fb2.png)

**Yours:**
![Screen Shot 2021-09-07 at 19 01 49](https://user-images.githubusercontent.com/22258274/132424899-0c7cb619-3afa-4a07-81ac-05b3b3f09b95.png)

#
**Mobile Size**
- It has a white background.

**Desktop Size**
- It has a dark background.

Reply:
This issue is well resolved! ✔️